### PR TITLE
fixed docs

### PR DESCRIPTION
**Pull request by the872**

### CODE REVIEW

File: examples/plot_poisson_2.py [hunk 1]
```diff
@@ -80,19 +81,27 @@
...
+#
+
+########################################################
 
 ########################################################
 
 z = np.linspace(0., 10., 100)
 qu = glm_poissonexp.qu(z)
-plt.plot(z, qu)
+plt.plot(z, qu, label='a')
```
Comment (jasmainak) [inline examples/plot_poisson_2.py]: can you remove the `plt.hold` in the next line? It's very MATLAB style. You don't need it in python

File: examples/plot_poisson_2.py [hunk 2]
```diff
@@ -80,19 +81,27 @@
...
 plt.hold
-plt.plot(z, np.exp(z))
+plt.plot(z, np.exp(z), label='b')
```
Comment (jasmainak) [inline examples/plot_poisson_2.py]: I still don't understand what the plot is supposed to communicate ...

File: examples/plot_poisson_2.py (hunk 1 above)
Comment (the872) [inline examples/plot_poisson_2.py]: Got it. I'll do that in a second.

File: examples/plot_poisson_2.py (hunk 2 above)
Comment (the872) [inline examples/plot_poisson_2.py]: I wasn't sure on what to properly label them as so I just assigned arbitrary values that can be easily fixed later.

**Comment by pavanramkumar** [comment]
thanks @the872, i'll merge this PR for now, but i can do some re-organizing tomorrow. see issue #106 for more potential docs-related contributions

**Pull request merged**